dbt/include/impala/macros/insertoverwrite.sql

```diff
@@ -17,7 +17,11 @@
     {%- set partition_cols = config.get('partition_by', validator=validation.any[list]) -%}
 
     {% if partition_cols is not none %}
-        {%- set partition_col = partition_cols[0] -%}
+        {% if partition_cols is string %}
+           {%- set partition_col = partition_cols -%}
+        {% else %}
+           {%- set partition_col = partition_cols[0] -%}
```

Comment:
If there are multiple columns in the partition by clause is the first column always chosen. What about the rest of the columns.